### ISSUE TITLE
Migrating away from Angular 1.x?

### ISSUE THREAD

**Issue by dwwinters** [lhncbc/lforms#21]
Are there any immediate plans to migrate lforms away from Angular 1.x?

**Comment by plynchnlm** [comment]
Yes.  I am about to start experimenting with transitioning to Web Components (with StencilJS).  But, although the plans are immediate, the timeline is not.  It is one of several things we are working on, and assuming the experiment is successful, it will still take a while to completely remove AngularJS.

**Comment by dwwinters** [comment]
Thanks for the response @plynchnlm!  I realize this is all conjecture, but do you envision an upgrade path for projects which have incorporated AngularJS versions of lforms?

**Comment by plynchnlm** [comment]
I would not expect the change to break any applications, unless said applications were relying on AngularJS packages bundled with the browser-ready package of LForms.  The API should not be afffected, and the purpose of moving to Web Components is that they (supposedly) compatible with existing major frameworks.
